addons/ingress-nginx/addon.rb

```diff
@@ -27,25 +27,25 @@
     )
   }
 
-  DEFAULT_BACKEND_ARM64_IMAGE = 'docker.io/kontena/pharos-default-backend-arm64:0.0.2'
-  DEFAULT_BACKEND_IMAGE = 'docker.io/kontena/pharos-default-backend:0.0.2'
-
   def image_name
     return config.default_backend[:image] if config.default_backend&.dig(:image)
 
-    if cpu_arch.name == 'arm64'
-      DEFAULT_BACKEND_ARM64_IMAGE
-    else
-      DEFAULT_BACKEND_IMAGE
-    end
+    "docker.io/kontena/pharos-default-backend-#{cpu_arch.name}:0.0.3"
```

Comment:
This conflicts with #655 ... maybe leave this out?

Reply:
Yeah, I'll refactor&rebase once #655 lands